src/Parser/TemplateParser.cs

```diff
@@ -14,17 +14,17 @@ public static PartList Parse(string template)
             return parser.Parse();
         }
 
-        private static readonly char[] HoleDelimiters = { ',', ':', '}' };
-        private static readonly char[] AlignmentDelimiters = { ':', '}' };
+        private static readonly char[] HoleDelimiters = { ':', '}' ,',' };
         private static readonly char[] TextDelimiters = { '{', '}' };
         private readonly string _template;
         private readonly int _length;
         private int _position;
         private int _holeIndex;
-        private PartList _parts;
+        private readonly PartList _parts;
 
         private TemplateParser(string template)
         {
+            _parts = new PartList();
```

Comment:
I think it's good practice that we don't get nullrefs in the code, so therefor it's in the ctor and not dependend of a method.

Too bad it's worse for the perf, but I guess creating a parser and don't used it, is bad